### ISSUE TITLE
bug(scope): version 2.1.0 still have 2.0.6 text

### ISSUE THREAD

**Issue by zinwelzl** [Security-Tools-Alliance/rengine-ng#83]
### Current Behavior

version 2.1.0 still have 2.0.6 text

### Expected Behavior

version 2.1.0 still have 2.0.6 text

### Steps To Reproduce

version 2.1.0 still have 2.0.6 text

### Environment

```markdown
- reNgine: 
- OS: 
- Python: 
- Docker Engine: 
- Docker Compose: 
- Browser:
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by yarysp** [comment]
Hi, you're right, I will correct this asap.
You're currently testing the 2.1.0 ?
Feel free to give us a lot of feedback 😉

**Comment by yarysp** [comment]
Resolved in branch release/2.1.0
https://github.com/Security-Tools-Alliance/rengine-ng/pull/1/commits/8bfda54996895f8238d2effd6a66318ea6479b3b
